src/Core/Form/FormHandler.php

```diff
@@ -84,11 +85,16 @@ public function getForm()
         }
 
         $this->formBuilder->setData($this->formDataProvider->getData());
-        $this->hookDispatcher->dispatchForParameters("display{$this->hookName}Form", [
-            'form_builder' => &$this->formBuilder
-        ]);
+        $this->hookDispatcher->dispatchForParameters(
+            "display{$this->hookName}Form",
```

Comment:
it's also an action hook as this is not about rendering a string content

Reply:
> it's also an action hook as this is not about rendering a string content

My point exactly